src/core/css/SearchResult.scss

```diff
@@ -34,10 +34,16 @@
   margin: 0;
   padding: 0;
   text-decoration: none;
+  width: calc(100% - 40px);
```

Comment:
For addon listing 
Calc(100% - 40px) - 38 px is fixed width `.SearchResult-icon` hence (100%-40px) for `.SearchResult-heading`